cql3/query_processor.hh

```diff
@@ -88,6 +88,17 @@ public:
 class cql_config;
 class query_options;
 class cql_statement;
+class query_processor;
+
+class wasm_context {
+    std::shared_ptr<rust::Box<wasmtime::Engine>> _engine;
+    std::optional<wasm::instance_cache> _instance_cache;
+    std::shared_ptr<wasm::alien_thread_runner> _alien_runner;
```

Comment:
We should use indirection to avoid making everyone know about rust::Box. @wmitros please follow up with dependency reduction.